ansible/roles/baselayout/vars/main.yml

```
@@ -64,7 +64,7 @@ packages: {
   ],
 
   fedora: [
-    'bzip2,ccache,gcc-c++,git,fontconfig,sudo',
+    'bzip2,ccache,gcc-c++,git,fontconfig,sudo,make',
```

could we add `ninja-build` as well?
```suggestion
    'bzip2,ccache,gcc-c++,git,fontconfig,sudo,make,ninja-build',
```

---

we could, but why? are we testing that now? and do you know for sure that's available on fedora30?